app/src/main/java/com/example/android/persistence/codelab/db/User.java

```diff
@@ -18,11 +18,14 @@
 
 import android.arch.persistence.room.Entity;
 import android.arch.persistence.room.PrimaryKey;
+import android.support.annotation.NonNull;
 
 
 @Entity
 public class User {
-    public @PrimaryKey String id;
+    public @PrimaryKey
```

Comment:
Keep the order like in the `Book` class: 
``` java
@PrimaryKey
@NonNull
public String id;
```

Reply:
Added files.